packages/commonwealth/server/workers/sitemap/createDatabasePaginator.ts

```diff
@@ -0,0 +1,204 @@
+import {
+  models,
+  ProfileAttributes,
+  ThreadInstance,
+} from '@hicommonwealth/model';
+import { QueryTypes } from 'sequelize';
+import { getThreadUrl } from 'utils';
```

Comment:
This import will break the execution of the build. This is the same import issue from your #7003 PR. You need to relatively import from `shared`.

Reply:
The 'utils' import? Can we block this via eslint? I think this is because of 'paths' ?

Reply:
Yes, it's because of `paths` and because https://github.com/hicommonwealth/commonwealth/pull/6796 hasn't been merged. If you can get these throw eslint errors that would be awesome but note it should only throw for these imports on the server not the client.